src/openrct2-ui/windows/Options.cpp

```diff
@@ -359,9 +362,10 @@ static rct_widget window_options_advanced_widgets[] = {
     { WWT_CHECKBOX,         1,  10,     299,    129,    140,    STR_ALWAYS_NATIVE_LOADSAVE,                 STR_ALWAYS_NATIVE_LOADSAVE_TIP },                   // Use native load/save window
     { WWT_DROPDOWN,         1,  165,    299,    145,    157,    STR_NONE,                                   STR_NONE },                                         // Autosave dropdown
     { WWT_BUTTON,           1,  288,    298,    146,    156,    STR_DROPDOWN_GLYPH,                         STR_AUTOSAVE_FREQUENCY_TIP },                       // Autosave dropdown button
-    { WWT_LABEL,            1,  23,     298,    165,    176,    STR_PATH_TO_RCT1,                           STR_PATH_TO_RCT1_TIP },                             // RCT 1 path text
-    { WWT_BUTTON,           1,  24,     289,    180,    193,    STR_NONE,                                   STR_STRING_TOOLTIP },                               // RCT 1 path button
-    { WWT_BUTTON,           1,  289,    299,    180,    193,    STR_CLOSE_X,                                STR_PATH_TO_RCT1_CLEAR_TIP },                       // RCT 1 path clear button
+    SPINNER_WIDGETS         (1,  165,    299,    165,    180,    STR_NONE,                                   STR_AUTOSAVE_AMOUNT_TIP ),                         // Autosave amount spinner
```

Comment:
These widgets do not appear for me. Have you made sure they fit in the window?

Minor change to improve reability (done everywhere else where `SPINNER_WIDGETS` is used:
```suggestion
    SPINNER_WIDGETS        (1,  165,    299,    165,    180,    STR_NONE,                                   STR_AUTOSAVE_AMOUNT_TIP ),                          // Autosave amount spinner
```

Reply:
Actually it seems the entry fits in my build:
![unbenannt](https://user-images.githubusercontent.com/1716810/50028873-57cc2d00-fff1-11e8-8dbb-941fd2a3b6ce.JPG)

Reply:
In the screenshot above, the field is a little too high, and the line spacing is off. Have a look at other spinners to see what (line) height they use.

Also, the decimals should be hidden, as the setting is an integer.

Reply:
> In the screenshot above, the field is a little too high, and the line spacing is off. Have a look at other spinners to see what (line) height they use.
> 
> Also, the decimals should be hidden, as the setting is an integer.

Fixed after removing the decimals.